config/settings.yml

```diff
@@ -12,6 +12,9 @@ BULK_METADATA:
   XML: 'MODS'
   ZIP: 'desc_metadata_download.zip'
 
+CHECKSUM_REPORT_JOB:
```

Comment:
Please lowercase these.  The config gem documents the correct usage as lowercase: https://github.com/railsconfig/config#embedded-ruby-erb  We have lowercase variables already in this file.  If we use uppercase the config gem ends up creating methods with the ruby convention for constants, which is confusing.

Reply:
sure.